test/tests/functional/pbs_job_array.py

```diff
@@ -82,6 +82,9 @@ def kill_and_restart_svr(self):
         except PbsServiceError as e:
             # The server failed to start
             raise self.failureException("Server failed to start:" + e.msg)
+        rv = self.is_server_licensed(self.server)
```

Comment:
Do you want to call self.server.isUp()?  It waits for the server to be fully up and responsive.

Reply:
I'll use isUp() call , But i need is_server_licensed ,  As all the test cases include server restart. It takes few secs time to show up in license_count , my jobs may not run even if i kick sched cycle . So i am waiting for licenses to show up.